common/static/js/vendor/jasmine-stealth.js

```diff
@@ -1,244 +1,214 @@
-// Generated by CoffeeScript 1.3.3
-
-/*
-jasmine-stealth 0.0.12
-Makes Jasmine spies a bit more robust
-site: https://github.com/searls/jasmine-stealth
-*/
-
-
+/* jasmine-stealth - 0.0.17
```

Comment:
Note to reviewers: I don't believe this NEEDED to be upgraded-- I tried upgrading it to see if it would fix issues. It seems worth checking in the upgrade though because it moves us to the last supported version of the library.